jobbergate-api/alembic/versions/20230801_071440--3bc095975b53_initial_migration_for_jobbergate_4.py

```diff
@@ -45,55 +45,59 @@ def upgrade():
     )
     op.create_table(
         "job_script_template_files",
-        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("parent_id", sa.Integer(), nullable=False),
```

Comment:
Side note: Is there a way to verify this on the API's test suite? All tests were passing, but the migration file was outdated.

Reply:
The unit tests don't use the migrations. They use `create_all()` to create the empty tables. Perhaps this is a good opportunity for an integration test?

Reply:
I'll keep an eye on it, but this shouldn't be a problem for this PR, so I'm resolving the comment.